ynr/apps/elections/uk/templates/includes/by-election-ctas.html

```diff
@@ -1,8 +1,9 @@
 {% if SHOW_BY_ELECTION_CTA %}
     <h2>By-elections</h2>
-
+<p>For all elections, including the May 2019 locals, see the <a href="{% url "election_list_view" %}">elections page</a>.</p>
```

Comment:
As we get nearer the time, it might be useful to have a special page for this as this page begins with by-elections in the past and shows the 2019 locals at the bottom.

Reply:
Yep – this involves changing the `FRONT_PAGE_CTA` setting away from by-elections to one of the other options. Not sure we have a _great_ one for locals. We have some stats and progress related ones, but generally making it easier to find what needs doing is a design thing I'd like to think about this year.